.github/workflows/bench.yml

```diff
@@ -105,19 +112,23 @@ jobs:
       - name: Convert for profiler.firefox.com
         run: |
           perf script -i perf.data -F +pid > transfer.perf &
-          perf script -i client/perf.data -F +pid > client.perf &
-          perf script -i server/perf.data -F +pid > server.perf &
-          wait
           mv flamegraph.svg transfer.svg
```

Comment:
As above, it might be better to have this file created in the right place to start with.